src/Pages/About/About.module.scss

```diff
@@ -0,0 +1,171 @@
+* {
+  background-color: #ffcb81;
+}
```

Comment:
Don't give background colour to * i.e. everything. Instead give it to the topmost class of your about page